src/argilla/training/setfit.py

```diff
@@ -100,6 +110,10 @@ def predict(self, text: Union[List[str], str], as_argilla_records: bool = True):
         Returns:
           A list of predictions
         """
+        if self._model is None:
+            self._logger.info("Using model without fine-tuning.")
```

Comment:
I think `self._logger.warn` is more appropriate here, would you agree?

Reply:
Agree